eventPage.js

```diff
@@ -40,7 +40,9 @@ let defaultSuggestionURL = '';
 
 function redirect(tab) {
   const url = dotComToDotDev(tab.url);
-  chrome.tabs.update({ url: url !== undefined ? url.toString() : root });
+  if(url !== undefined) {
+    chrome.tabs.update({ url });
+  }
```

Comment:
Thank you for the fix! This should really be 

```js
  chrome.tabs.update({ url: url !== undefined ? url.toString() : INSIDERS_VSCODE_DEV });
```

If you don't want to update the PR, please just let me know and I can pick it up from here.